internal/languages/language.go

```diff
@@ -1,3 +1,9 @@
+// This file wraps the logic of go-enry (https://github.com/go-enry/go-enry) to support additional languages.
+// go-enry is based off of a package called Linguist (https://github.com/github/linguist)
+// and sometimes programming languages may not be supported by Linguist
```

Comment:
Let's also mention how the logic is duplicated from `sourcegraph/sourcegraph`, and how we plan to unify this in the future.